lansa-vmss-windows-autoscale-sql-database/azuredeploy.json

```diff
@@ -0,0 +1,809 @@
+{
+   "$schema": "http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json",
```

Comment:
would be best to just be 2015-01-01 (without the "-preview"); sorry I missed this before